docs/install.md

```diff
@@ -0,0 +1,76 @@
+## Installation
+
+Following ways are supported to run Cerberus:
+
+- Standalone python program through Git or python package
+- Containerized version using either Podman or Docker as the runtime
+- Kubernetes or OpenShift deployment
+
+**NOTE**: Only OpenShift 4.x versions are supported.
+ 
+
+## Git
+```
+$ git clone https://github.com/openshift-scale/cerberus.git
+```
+
+### Install the dependencies
+```
+$ pip3 install -r requirements.txt
+```
+
+### Configure and Run
+Setup the [config](https://github.com/openshift-scale/cerberus/tree/master/config) according to your requirements. Information on the available options can be found at [usage](usage.md).
+
+#### Run
+```
+$ python3 start_cerberus.py --config <config_file_location>
+```
+
+**NOTE**: When config file location is not passed, default [config](https://github.com/openshift-scale/cerberus/tree/master/config) is used.
+
+
+## Python Package
+Cerberus is also available as a python package to ease the installation and setup.
+
+To install the lastest release:
+
+```
+$ pip3 install cerberus-client`
+```
+
+### Configure and Run
+Setup the [config](https://github.com/openshift-scale/cerberus/tree/master/config) according to your requirements. Information on the available options can be found at [usage](usage.md).
+
+#### Run
+```
+$ cerberus_client -c <config_file_location>`
+```
+
+**NOTE**: When config_file_location is not passed, default [config](https://github.com/openshift-scale/cerberus/tree/master/config) is used.
+**NOTE**: It's recommended to run Cerberus either using the containerized  or github version to be able to use the latest enhancements and fixes.
+
+## Containerized version
+
+Assuming that the latest docker ( 17.05 or greater with multi-build support ) is intalled on the host, run:
```

Comment:
Perhaps remove "latest docker"

"Assuming docker 17.05 or greater with multi-build support..."

Reply:
Fixed it, thanks.